bigquery/test/datasets.test.js

```diff
@@ -124,27 +117,24 @@ describe('bigquery:datasets', function () {
 
       sample.program.listDatasets(projectId, callback);
 
-      assert(sample.mocks.bigquery.getDatasets.calledOnce, 'method called once');
-      assert.equal(sample.mocks.bigquery.getDatasets.firstCall.args.length, 1, 'method received 1 argument');
-      assert(callback.calledOnce, 'callback called once');
-      assert.equal(callback.firstCall.args.length, 2, 'callback received 2 arguments');
-      assert.ifError(callback.firstCall.args[0], 'callback did not receive error');
-      assert.strictEqual(callback.firstCall.args[1], sample.mocks.datasets, 'callback received result');
-      assert(console.log.calledWith('Found %d dataset(s)!', sample.mocks.datasets.length));
+      assert.equal(sample.mocks.bigquery.getDatasets.calledOnce, true);
+      assert.deepEqual(sample.mocks.bigquery.getDatasets.firstCall.args.slice(0, -1), []);
+      assert.equal(callback.calledOnce, true);
+      assert.deepEqual(callback.firstCall.args, [null, sample.mocks.datasets]);
+      assert.equal(console.log.calledOnce, true);
+      assert.deepEqual(console.log.firstCall.args, ['Found %d dataset(s)!', sample.mocks.datasets.length]);
     });
 
     it('should handle error', function () {
-      var error = 'error';
+      var error = new Error('error');
```

Comment:
Yes, it's definitely more idiomatic for errors to actually be instances of `Error`.